### DISCUSSION TITLE
random poor test results using the cli

### DISCUSSION THREAD

**Discussion by skyeci-draytek** [alexjustesen/speedtest-tracker#1249] [Q&A]
hi,

Just wondered if others of you see random poor upload or download results when using a few self chosen servers. I find I see this issue a couple of times a day on different servers. Is this a cli issue or a wider problem. I compared this to another cli tool I found which does not use the speedtest cli and the results are consistent with no poor results.. Im on a 2gb fibre symmetrical connection so it should be reliable.

Just interested to hear others experiences.

thanks

**Comment by seh2000** [comment]
Not really, once in a while I do see a "poor" result, but in my opinion this is not related to the application.
I tested/tried servers available in my area and settled for the server that gave the more steady results. Tried each server via the app here and via Ookla. I am on a fiber connection having 1G DL and 0.1G UL. I am running the test in 30 min intervals. 

![Screenshot 2024-02-26 115900](https://github.com/alexjustesen/speedtest-tracker/assets/74658646/7c6c21c4-8962-4376-8fa4-1cf4f058108c)

**Reply by skyeci-draytek** [reply]
I understand what you are saying but when I use one of my isp servers it occasionally throws a curve ball of 100mb upload when I'm on a 2gb symmetrical ..just a bit odd
